dictionaries/software-terms/dict/softwareTerms.txt

```diff
@@ -193,6 +193,7 @@ SoftLayer
 Solaris
 Solr
 Sparc
+SquashFS
```

Comment:
This is an auto generated file from the source file. Please add it to [`software-terms/src/coding-terms.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/coding-terms.txt).

Reply:
Updated. Although kind of a strange place to put it in, given it has little to do with coding. Maybe a dedicated file for filesystems would make more sense, or `something-os` similarly to `networking-os`?